.github/workflows/plugin_install.yml

```diff
@@ -1,36 +1,120 @@
-name: Plugin Install
-
-on: [push, pull_request, workflow_dispatch]
-
-jobs:
-  plugin-install:
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [windows-latest, ubuntu-latest]
-        jdk: [11, 17]
-    runs-on: ${{ matrix.os }}
-    steps:
-      - name: Set up JDK
-        uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.jdk }}
-
-      - name: Checkout Branch
-        uses: actions/checkout@v2
-
-      - name: Assemble target plugin
-        run: ./gradlew assemble
-
-      - name: Run Opensearch with A Single Plugin
-        uses: ./.github/actions/start-opensearch-with-one-plugin
-        with:
-          opensearch-version: 3.0.0
-          plugin-name: opensearch-security
-          plugin-start-script: install_demo_configuration
-          docker-host-plugin-zip: security-plugin.zip
-
-      - name: Run sanity tests
```

Comment:
Why did we remove the sanity test step?

Reply:
So I realized that during the conflict resolution it messed up some of the files so I think that is why things looked so weird to you before. I readded this and fixed the copying over of the action to the workflow.